### ISSUE TITLE
1.4 billion password breach compilation wordlist

### ISSUE THREAD

**Issue by riramar** [danielmiessler/SecLists#154]
How about include this password list?

https://gist.github.com/scottlinux/9a3b11257ac575e4f71de811322ce6b3

**Comment by mazen160** [comment]
This would be quite difficult to include. The size of the wordlist is 4.5GB+. May be referencing a link to it?

**Comment by hinell** [comment]
@mazen160 
Here is one 1gb compressed and 4+ expanded:
https://files.vkk.me/storage/breachcompilationuniq.txt.7z
I haven't checked it yet.
Initially it was posted on the [reddit](https://www.reddit.com/r/netsec/comments/7kqpx9/recent_14_billion_password_breach_compilation_as/drh79oi/)

**Comment by Al-tamimi** [comment]
question, 
how to use file to crack password, through terminal? or what 



thank you

**Comment by roybollinger** [comment]
What do you think of http://leakprobe.net would anyone use this service for leaked passwords?

**Comment by g0tmi1k** [comment]
You can only do 100MB on github ~ https://help.github.com/en/articles/conditions-for-large-files

**Comment by ArtemSer** [comment]
It can be split into many part-files?

**Comment by BaRRaKudaRain** [comment]
This is and idea!

--
---------------------------
An outsider, the BaRRaKudaRain

-----Original Message-----

**Comment by cwalkbb** [comment]
Wow

**Comment by dido4free** [comment]
> 
> 
> What do you think of http://leakprobe.net would anyone use this service for leaked passwords?

I have rechecked above service. Most of leaked passwds are old or too old (for example 5-9 years old). But there are about 10-20% actual passwords (I have checked about 50 users). 
And you will not have ability to use that leaded passwds because of many www see your wrong location with e-mail owner. 
So maybe there are other better base with leaked passwds?
